libsolidity/interface/CompilerStack.cpp

```diff
@@ -359,6 +361,14 @@ bool CompilerStack::parse()
 			{
 				solAssert(!import->path().empty(), "Import path cannot be empty.");
 
+				auto it = solidity::solstdlib::sources.find(import->path());
```

Comment:
Does that mean you can import a stdlib file without `pragma stdlib`? Do we want that?

Also, do we check for clashes between local files and stdlib files?

Reply:
Yeah, there's no checks at the moment of any kind - this is really just a step in the 'right' direction compared to the previous approach which would always include the `stdlib` sources to be parsed, regardless of whether they were explicitly `import`ed or not; this was the cause for the vast majority of the test failures, which I wanted to get rid of in order to have a better picture of what to prioritise.

But yeah, once I figure out the bytecode compare failures on Windows, I'll focus on the rest. There's still quite a bit of stuff to do; e.g. `cryptography.sol` (ex `precompiles.sol`) is a working, but not a particularly robust implementation, at least compared to what the IR Generator (for statements) produces.